Signal/src/ViewControllers/ConversationView/Cells/OWSMessageCell.m

```diff
@@ -130,7 +131,6 @@ - (void)commontInit
     self.textView = [UITextView new];
     // Honor dynamic type in the message bodies.
     self.textView.font = [self textMessageFont];
-    self.textView.font = [UIFont ows_regularFontWithSize:16.f];
```

Comment:
I don't know if this was rebase error, but this was probably breaking dynamic type handling.